C/manual/tips-appendix.xml

```diff
@@ -155,7 +155,7 @@
                     </listitem>
 
                     <listitem>
-                      <para>To use him also with <command>gnc-fq-<replaceable>*</replaceable></command> commands you should set
+                      <para>To use him also with <command>gnucash-cli --quotes dump</command> commands you should set
                         an <emphasis>environment variable</emphasis>
                         <envar>ALPHAVANTAGE_API_KEY</envar>. Please consult the manual of your OS or
                         command shell.
```

Comment:
This is no longer necessarily true: `gnucash-cli -Q` will query gsettings for the API key so it's no longer necessary to set an environment variable too.

Reply:
So it should be set in Preferences.